src/test/regress/sql/create_role_propagation.sql

```diff
@@ -107,7 +107,7 @@ RESET ROLE;
 
 SET citus.enable_create_role_propagation TO ON;
 
-GRANT dist_role_3 TO non_dist_role_3;
+GRANT dist_role_3 TO non_dist_role_3 granted by postgres;
```

Comment:
What does this test? Isn't the current user already `postgres`? And we are not looking at a propagated query or something like that.

Reply:
Changed to test_admin_role